analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -93,7 +93,8 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
         private const val MAX_DISK_CACHE_ENTRY_AGE_SECONDS = 6 * 60 * 60
 
         // See http://maven.apache.org/pom.html#SCM.
-        val SCM_REGEX = Pattern.compile("scm:(?<type>[^:]+):(?<url>.+)")!!
+        val SCM_REGEX = Pattern.compile("scm:(?<type>[^:@]+):(?<url>.+)")!!
+        val USER_HOST_REGEX = Pattern.compile("scm:(?<user>[^@]+)@(?<host>.+):(?<url>.+)")!!
```

Comment:
Instead of `?<host>.+`, should we say `?<host>[^:]+` (analogous to `user` and `@`)?

Reply:
Done.